app/controllers/projects_controller.rb

```diff
@@ -56,7 +56,7 @@ def create
       redirect_to "/projects"
     else
       flash[:error] = @project.errors.full_messages
-      redirect_back(fallback_location: "projects/new")
+      render :new
```

Comment:
you should probably do `flash.now[:error] = ...` if you render instead of redirect

Reply:
@rishijain did you see this comment from Ariel?

Reply:
@arielj are you happy with this [answer](https://github.com/fastruby/points/pull/248#discussion_r872256868)?